site/content/docs/main/config/overload-manager.md

```diff
@@ -0,0 +1,30 @@
+# Overload Manager
+
+Envoy uses heap memory when processing requests.
+When the system runs out of memory or memory resource limit for the container is reached, Envoy process is terminated abruptly.
+To avoid this, Envoy [overload manager][1] can be enabled.
+Overload manager controls how much memory Envoy will allocate at maximum and what actions it takes when the limit is reached.
+
+Overload manager is disabled by default.
+It can be enabled at deployment time by using `--overload-max-heap=[MAX_BYTES]` command line flag in [`contour bootstrap`][2] command.
+The bootstrap command is executed in [init container of Envoy pod][3] to generate initial configuration for Envoy.
+To enable overload manager, modify the deployment manifest and add for example `--overload-max-heap=2147483648` to set maximum heap size to 2 GiB.
+The appropriate number of bytes can be different from system to system.
+
+After the feature is enabled, following two overload actions are configured to Envoy:
+
+* Shrink heap action is executed when 95% of the maximum heap size is reached.
+* Envoy will stop accepting requests when 99% of the maximum heap size is reached.
```

Comment:
off-by-one 😄  
```suggestion
* Envoy will stop accepting requests when 98% of the maximum heap size is reached.
```

Reply:
Thanks, apparently I'm not any good with percentages :blush: Corrected !